config/api.yml

```diff
@@ -647,6 +647,46 @@
       :get:
       - :name: read
         :identifier: load_balancer_show
+  :alerts:
+    :description: Alerts
+    :identifier: alert
+    :options:
+    - :collection
+    - :subcollection
+    :verbs: *g
+    :klass: MiqAlertStatus
+    :subcollections:
+    - :alert_actions
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: alert_status_show_list
+    :resource_actions:
+      :get:
+      - :name: read
+        :identifier: alert_status_show
+  :alert_actions:
+     :description: Alert Actions
+     :identifier: alert_action
+     :options:
+     - :subcollection
+     - :collection
+     :verbs: *gp
+     :klass: MiqAlertStatusAction
+     :collection_actions:
+       :get:
+       - :name: read
+         :identifier: alert_status_action_show_list
+       :post:
+       - :name: create
+         :identifier: alert_status_action_new
+     :subcollection_actions:
+       :get:
+       - :name: read
+         :identifier: alert_status_action_show_list
+       :post:
+       - :name: create
```

Comment:
does it make sense to allow creates as both /api/alert_actions as well as /api/alerts/:id/alert_actions ?  Doesn't the former need a relationship to the /api/alerts resource ?